src/ext/java/org/opentripplanner/ext/reportapi/resource/ReportResource.java

```diff
@@ -20,6 +26,9 @@
 @Produces(MediaType.TEXT_PLAIN)
 public class ReportResource {
 
+  @Nullable
+  private static CachedValue<GraphStats> cachedStats;
```

Comment:
```suggestion
/** Since the computation is pretty expensive only allow it every 5 minutes */
  private static CachedValue<GraphStats> cachedStats = new CachedValue<>(Duration.ofMinutes(5));;
```